RealifeTech-CoreSDK/GraphQL/GraphQLFiles/sell_getMyPaymentSources.graphql

```diff
@@ -0,0 +1,25 @@
+query getMyPaymentSources($pageSize: Int!, $page: Int! = 1) {
+  getMyPaymentSources(page: $page, pageSize: $pageSize) {
+    edges {
+      ...paymentSourceDetails
+    }
+  }
+}
+
+fragment paymentSourceDetails on PaymentSource {
```

Comment:
I'd prefer putting the fragments in another file for the clear files structure, but it's fine to leave it as it is.

Reply:
I agree with you. And I can do this as it won't take too much time. :)